ext/openssl/ossl_config.c

```diff
@@ -305,18 +305,16 @@ static IMPLEMENT_LHASH_DOALL_ARG_FN(dump_conf_value, CONF_VALUE, VALUE)
  *
  * Gets the parsable form of the current configuration.
  *
- * Given the following configuration being created:
+ * Given the following configurating file being loaded:
```

Comment:
How about: 

```suggestion
 * Given the following configuration file being loaded:
```

Reply:
@olleolleolle Thanks for your suggestion.
I chose the same terminology as in the other example, but If you find this unnatural, I am willing to accept your suggestion.

https://github.com/ruby/openssl/blob/8367b16642f3a325d2344df6d9bcff6dd4f11393/ext/openssl/ossl_config.c#L170
https://github.com/ruby/openssl/blob/8367b16642f3a325d2344df6d9bcff6dd4f11393/ext/openssl/ossl_config.c#L209

Reply:
Ah, thanks, I would change those too, I can make a separate PR.

Reply:
@olleolleolle I'm sorry, I was late due to my misunderstanding.
I pushed "Commit suggestion" button. Thank you.